src/compressor.h

```diff
@@ -118,4 +118,28 @@ class CTxOutCompressor
     }
 };
 
+/** wrapper for CTzeOut that provides a more compact serialization */
+class CTzeOutSer
```

Comment:
```suggestion
class CTzeOutCompressor
```
(for consistency with `CTxOutCompressor`)